packages/compartment-mapper/src/link.js

```diff
@@ -60,6 +60,8 @@ const has = (object, key) => apply(hasOwnProperty, object, [key]);
  */
 const extensionImpliesLanguage = extension => extension !== 'js';
 
+const urlish = /^[a-z][a-z0-9+\-.]*:/;
```

Comment:
This syntax is subject to an external design constraint; let's please cite it:

```suggestion
const urlish = /^[a-z][a-z0-9+\-.]*:/; // cf. section 4.3 of RFC 3986 Scheme URI Generic Syntax
```

optionally include https://www.rfc-editor.org/rfc/rfc3986#section-4.3